.gitlab-ci.yml

```diff
@@ -0,0 +1,95 @@
+before_script:
```

Comment:
When does this run on gitlab?

Reply:
I have my own GitLab farm that I use for personal stuff and it was easier to run it there, additionally allowed me to figure out how to write a GitLab script for libraries. Right now it's just building the example project.